Sources/Turf/GeoJSON.swift

```diff
@@ -3,103 +3,60 @@ import Foundation
 import CoreLocation
 #endif
 
-public enum FeatureType: String, Codable {
-    case feature = "Feature"
-    case featureCollection = "FeatureCollection"
-}
-
-struct FeatureProxy: Codable {
-    public var type: FeatureType
-        
-    private enum CodingKeys: String, CodingKey {
-        case type
-    }
+/**
+ A [GeoJSON object](https://datatracker.ietf.org/doc/html/rfc7946#section-3) represents a Geometry, Feature, or collection of
+ Features.
+ */
+public enum GeoJSONObject: Equatable {
+    /**
+     A [Geometry object](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1) represents points, curves, and surfaces in coordinate space.
+     
+     - parameter geometry: The GeoJSON object as a Geometry object.
+     */
+    case geometry(_ geometry: Geometry)
```

Comment:
Is the parameter naming just to benefit documentation?

Reply:
The parameter name also appears in the completion token when code-completing the case.